specification/peering/resource-manager/Microsoft.Peering/preview/2019-08-01-preview/peering.json

```diff
@@ -0,0 +1,2448 @@
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "The name of the peering SKU.",
+          "enum": [
+            "Basic_Exchange_Free",
+            "Basic_Direct_Free",
+            "Premium_Direct_Free",
+            "Premium_Exchange_Metered",
+            "Premium_Direct_Metered",
+            "Premium_Direct_Unlimited"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "name",
+            "modelAsString": true
+          }
+        },
+        "tier": {
+          "description": "The tier of the peering SKU.",
+          "enum": [
+            "Basic",
+            "Premium"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "tier",
+            "modelAsString": true
+          }
+        },
+        "family": {
+          "description": "The family of the peering SKU.",
+          "enum": [
+            "Direct",
+            "Exchange"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "family",
+            "modelAsString": true
+          }
+        },
+        "size": {
+          "description": "The size of the peering SKU.",
+          "enum": [
+            "Free",
+            "Metered",
+            "Unlimited"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "size",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "PeeringProperties": {
+      "description": "The properties that define connectivity to the Microsoft Cloud Edge.",
+      "type": "object",
+      "properties": {
+        "direct": {
+          "$ref": "#/definitions/PeeringPropertiesDirect",
+          "description": "The properties that define a direct peering."
+        },
+        "exchange": {
+          "$ref": "#/definitions/PeeringPropertiesExchange",
+          "description": "The properties that define an exchange peering."
+        },
+        "peeringLocation": {
+          "description": "The location of the peering.",
+          "type": "string"
+        },
+        "provisioningState": {
+          "description": "The provisioning state of the resource.",
+          "enum": [
+            "Succeeded",
+            "Updating",
+            "Deleting",
+            "Failed"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "provisioningState",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "PeeringPropertiesDirect": {
+      "description": "The properties that define a direct peering.",
+      "type": "object",
+      "properties": {
+        "connections": {
+          "description": "The set of connections that constitute a direct peering.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DirectConnection"
+          }
+        },
+        "useForPeeringService": {
```

Comment:
>useForPeeringService [](start = 9, length = 20)

could it be inUseWithPeeringService or usedWithPeeringService ?
useForPeeringService feels more like customer opt-in whether to use or not.. But description is a bit different..

Reply:
nit nit - may word is like 'to be used with peering service' ,  instead of 'peering is used for peering service'.. 

---
In reply to: [316866687](https://github.com/Azure/azure-rest-api-specs/pull/6886#discussion_r316866687) [](ancestors = 316866687)

Reply:
We already have this variable at the properties level and we are adding it to the connection level.

Reply:
Peering service is the name of our service we are marketing to customers. ISPs can offer peering service to those business customers. A peering whether it uses our peering service is still a peering.